### PR TITLE
Da Grand Soap Resprite

### PR DESCRIPTION
**Pull request by TheRealScarHomie**
## About The Pull Request
Resprites the current soulful soap sprites to something more mediocre, also does some changes to the OMEGA SOAP, to make it add XP to Cleaning and Slip for a little longer (Due to the extreme lengths you have to go to make this, it is currently a huge let down.)

**SOAP**

alright third iteration here, downvotes are from an old edition BUT ITS BEEN REFINED

image

## Why It's Good For The Game
Current soap sprites are coder sprites and Omega Soap is a cool Sekrit that literally has 0 benefit apart from that "it clean fast", which including the fact that it dosent even add to cleaning XP (Even though it is the holy soap of only the most Robust Jannys) makes it not really worth the 30min it takes to make.

## Changelog
:cl:
add: Added new Soap sprites
tweak: Tweaked Slip and XP values for Omegasoap
sound: Added a holy sound on creation
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
i wouldnt say the current soap looks bad / like codersprites and i dont think these are necesarily a huge improvement

**Comment by Jared-Fogle** [comment]
they look like plates

**Comment by Angustmeta** [comment]
They kinda look like resprites of the serving tray. And I don't see how the word "omega" can be associated with the pride flag.

**Comment by WJohn** [comment]
These look much worse, in my opinion. They're like flat oval discs.

**Comment by optimumtact** [comment]
What do they look like IN the game, why do people always leave that screenshot out.

**Comment by itseasytosee** [comment]
I feel obligated to approve of any PR with an omega soap buff.

**Review by itseasytosee — APPROVED** [review]

**Comment by Doosteroni** [comment]
Pride soap? Unbased

**Comment by scarwolff** [comment]
@optimumtact 
image

**Comment by itseasytosee** [comment]
Modeled after the kitchen tray I see

**Comment by Ziiro** [comment]
They look like nice sprite work but I'm not a fan of them as a soap sprite. They do really look like serving trays.

**Comment by scarwolff** [comment]
any feedback on how I can "un-serving tray them?" Literally whammed these out in like 20min lmao, constructive critism is always welcome.

**Comment by scarwolff** [comment]
image
maybe a style something for akin to this?

**Comment by Qustinnus** [comment]
image


maybe like this

**Comment by scarwolff** [comment]
lmao I know that was a backhanded remark but check this out
image

**Comment by itseasytosee** [comment]
Looks better to me.

**Comment by scarwolff** [comment]
image

**Comment by scarwolff** [comment]
image

**Comment by Tharcoonvagh** [comment]
LEAVE MY SYND SOAP ALONE YOU BLANKET-HUGGING TABLEHOOTER

**Comment by trerri** [comment]
The original colors were good

**Comment by scarwolff** [comment]
yeah i mean last time I checked rainbow /= gay, its just a nice blend of colours

**Comment by wesoda25** [comment]
I just think more vibrant colors are neat.

**Comment by Angustmeta** [comment]
For some reason I always envisioned the blue galaxy brain meme when I thought of the omega soap. 

The syndie soap dark red text on dark red background is a bit hard to make out.

**Comment by Winterous** [comment]
Just too thin. Soap is THICK, soap is CHUNKY, soap requires you to really open your mouth to take a bite. These things are dinner plates!
Also if you're going to change soap sprites, the first thing I would do is make them smaller; they are GIGANTIC sprites for some reason.

**Comment by scarwolff** [comment]
im trying to homage to the original sprites with these designs otherwise I wouldve actually done some spriting and made new tiny round/squareish bars of soap, but that just aint TG, that aint soul, this is the look this server sticks with, something to think on (also tiny soap durr hurr hiding under paper).

**Comment by wesoda25** [comment]
I think they would look better if you shaved off the bottom row of pixels

**Comment by Winterous** [comment]
No don't shave the bottom pixel off, add a few more, they're too thin!

**Comment by scarwolff** [comment]
image
do these soaps look more edible

**Comment by SweptWasTaken** [comment]
So is the rainbow one a gay bar?

**Comment by scarwolff** [comment]
its just colourful, once again rainbow /= gay

**Comment by Winterous** [comment]
Beautiful, love it.
Only remaining criticism is that maybe the omega should be a gradient, rather than hard stripes. Or hell, the stripes could move like some sort of strobe light.
It does have the power to alter the time-space continuum after all, epileptic seizures are hardly beyond its abilities.

**Comment by scarwolff** [comment]
Sprite-0004

homosexual bar has been SCRUBBED (also, why is there a merge conflict now?)

**Comment by Qustinnus** [comment]
https://www.youtube.com/watch?v=IslF_EyhMzg

**Comment by SweptWasTaken** [comment]
image

**Comment by Twaticus** [comment]
while i do think these could use a resprite, our soaps are pretty classic which means any changes need to be a noticeable improvement and i dont think these fit the bill

**Comment by scarwolff** [comment]
how can I improve them so they fit "the bill" then, instead of immediately shutting down the PR give me advice and we can make them fit the bill (without a full soulsucking resprite preferably)

File: code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -85,6 +85,22 @@
...
+		if(do_after(user, clean_speedies, target = target))
+			to_chat(user, "<span class='notice'>You scrub \the [target.name] out.</span>")
+			var/obj/effect/decal/cleanable/cleanies = target
+			user?.mind.adjust_experience(/datum/skill/cleaning, max(round(cleanies.beauty/CLEAN_SKILL_BEAUTY_ADJUSTMENT,1),0)) //If someone has put the EFFORT in to aquire such HOLY soap, then they should at least get XP from it.
+			qdel(target)
+			decreaseUses(user)
+
+/obj/item/soap/omega/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/slippery, 150) //to take 40 minutes of dedication to make yould think it would slip just a lil more
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/clown_items.dm:102]: time define?

Comment (scarwolff) [reply code/game/objects/items/clown_items.dm:102]: same format as the one before it

**Comment by ExcessiveUseOfCobblestone** [comment]
The game changes should be separated assuming someone wants them.

**Comment by scarwolff** [comment]
I mean this PR was in majority just an Omega Soap overhaul (because it looks and functions like ass)

**Comment by scarwolff** [comment]
could make a seperate PR scrapping 40 mins of work and just keeping the Omega soap minor change and reskin?

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (Qustinnus) [reply code/game/objects/items/clown_items.dm:102]: just because older code is bad doesnt mean yours should be

**Comment by ExcessiveUseOfCobblestone** [comment]
sure if twat is ok with the sprite

**Comment by Doosteroni** [comment]
> its just colourful, once again rainbow /= gay

> homosexual bar has been SCRUBBED

Is it meant to be gay or not? YOU ARE SENDING MIXED SIGNALS!

**Comment by SpaceManiac** [comment]
Closing as stale.

Please split the functional changes and the sprite changes into two separate PRs so they can be considered on their merits separately.

**Pull request closed without merging**